pkg/lightning/tikv/local_sst_writer_test.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2024 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tikv
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+	"testing"
+
+	"github.com/cockroachdb/pebble"
+	"github.com/cockroachdb/pebble/sstable"
+	"github.com/cockroachdb/pebble/vfs"
+	"github.com/google/uuid"
+	rockssst "github.com/lance6716/pebble/sstable"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	sortedKVs = [][2][]byte{
+		{[]byte("a"), []byte("1")},
+	}
+	ts uint64 = 1
+)
+
+func TestGRPCWriteToTiKV(t *testing.T) {
+	t.Skip(`This is a manual test. You can use tiup playground and run this test.
+After the test is finished, find the SST files in the import directory of the TiKV node.`)
+
+	ctx := context.Background()
+	pdAddrs := []string{"127.0.0.1:2379"}
+
+	metas, err := write2ImportService4Test(ctx, pdAddrs, sortedKVs, ts)
+	require.NoError(t, err)
+	for _, meta := range metas {
+		t.Logf("meta UUID: %v", uuid.UUID(meta.Uuid).String())
+	}
+}
+
+func TestPebbleWriteSST(t *testing.T) {
+	//t.Skip("this is a manual test")
+
+	sstPath := "/tmp/test-write.sst"
+	writer, err := newWriteCFWriter(sstPath, ts, &rockssst.Identity{
```

Comment:
So the file created by `newWriteCFWriter` represents the TiKV's output?

Reply:
that file will replace TiKV's output in future. Currently we just compare the file content